docs/swagger.yaml

```diff
@@ -204,96 +204,183 @@ components:
       x-examples:
         example-1:
           imageId: laptop_image4433
+    ImageId:
+      title: ImageId
+      type: object
+      properties:
+        id:
+          type: string
+          pattern: '^[0-9A-Za-z-_\.]+$'
+          maxLength: 128
+      required:
+        - id
+    CategoryEnum:
+      type: string
+      title: CategoryEnum
+      enum:
+        - general
+        - receipt
+        - orderConfirmation
+        - delivery
+        - ticket
+        - booking
+    Receipt:
+      title: Receipt
+      type: object
+      properties:
+        orderLines:
+          type: array
+          minItems: 1
+          items:
+            $ref: '#/components/schemas/OrderLine'
+        bottomLine:
+          $ref: '#/components/schemas/BottomLine'
+      required:
+        - orderLines
+        - bottomLine
     OrderLine:
       title: OrderLine
       type: object
       description: 'All numbers are interpreted as integers with two trailing zeros. ex: 10.00 = 1000'
       properties:
-        amountExcludingTax:
+        totalAmount:
           type: integer
+          description: Total amount of the order line
           nullable: true
-        amountIncludingTax:
-          type: integer
-          nullable: true
-        description:
-          type: string
-        id:
-          type: string
-          description: The product ID
-        productUrl:
-          type: string
-          nullable: true
-        quantity:
+        totalAmountExcludingTax:
           type: integer
+          description: Total amount of order line excluding tax
           nullable: true
-        taxAmount:
+        totalTaxAmount:
           type: integer
+          description: Total tax amount for the order line
           nullable: true
         taxPercentage:
           type: integer
+          description: Tax percantage for the order line
           nullable: true
+        currency:
+          $ref: '#/components/schemas/CurrencyEnum'
+        name:
+          type: string
+          description: Name of the product in the order line
+        id:
+          type: string
+          description: The product ID
+        unitInfo:
+          $ref: '#/components/schemas/UnitInfo'
         discount:
           type: integer
+          description: Total discount for the order line
           nullable: true
         quantityUnit:
           $ref: '#/components/schemas/QuantityUnitEnum'
+        productUrl:
+          type: string
+          description: ''
+          nullable: true
+        unitPrice:
```

Comment:
should this be removed since we have unitInfo?

Reply:
Yes it should, my bad